js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx

```diff
@@ -0,0 +1,28 @@
+import { Alert, Button, Center, Text, VStack } from "@chakra-ui/react";
+import React from "react";
+import { useGraph } from "./GraphContext";
+
+export const SmartGraphEmptyState = () => {
+    const { graphName } = useGraph();
+
+    return (
+        <Center h='100%'>
+            <Alert width="auto" background="gray.300">
```

Comment:
we can remove the background color here, also can just use a regular box, Alert not needed

Reply:
Done.